### ISSUE TITLE
Disable createDbSubnet and enable createParameters

### ISSUE THREAD

**Issue by danielstandby** [smoketurner/serverless-vpc-plugin#534]
If I set `createDbSubnet` to `false`, and `createParameters` to `true`, `sls deploy` fails with this message:
```
An error occurred: ParameterDBSubnets - Property validation failure: [Length of value for property {/Value} is less than minimum allowed length {1}].
```

It seems you're trying to publish DBSubnet parameter even if `createDbSubnet` is set to `false`

**Comment by Alejandro287** [comment]
Same thing happened to me. Is there any update about this?

**Comment by jplock** [comment]
Sorry, haven't gotten a chance to take a look yet.

**Comment by jplock** [comment]
@Alejandro287 @danielstandby I just published `v1.0.3` to NPM.  Can you see if that version fixes this?

**Comment by Alejandro287** [comment]
It is working for me. Thank you for your help !
